mypy/nodes.py

```diff
@@ -1651,7 +1651,7 @@ def accept(self, visitor: ExpressionVisitor[T]) -> T:
         return visitor.visit_str_expr(self)
 
 
-def is_StrExpr_list(seq: list[Expression]) -> TypeGuard[list[StrExpr]]:
+def is_StrExpr_list(seq: list[Expression]) -> TypeGuard[list[StrExpr]]:  # type: ignore[typeguard-subtype]
```

Comment:
why is this typeguard wrong

Reply:
`list` is invariant

Reply:
should it be updated to take a `Sequence` instead or something? looks like that variable name is `seq` anyway, which i assume means sequence

Reply:
Not my code + the use site would be never anyway.